### DISCUSSION TITLE
particle trajectories

### DISCUSSION THREAD

**Discussion by MaryamBC** [DualSPHysics/DualSPHysics#242] [DUALSPHYSICS_V5.2]
Dear DualSPHysics Users,

I hope this message finds you well. I recently noticed that version 5.2 includes a new post-processing tool called TracerParts, which allows us to trace particle trajectories. However, I couldn't find detailed documentation on how to use this feature.

Could anyone let me know which examples incorporate this tool? Understanding its application through specific examples would greatly help me grasp how to effectively use it.

Thank you for your assistance!

**Comment by aepoc** [comment]
I would like to hear more on this as well!

**Comment by alexbexe** [comment]
You can find the syntaxis in the following examples:

DualSPHysics_v5.2\examples\main\01_DamBreak\wCaseDambreakVal2D
DualSPHysics_v5.2\examples\main\08_WavesFlap\wCaseFlapBeach
DualSPHysics_v5.2\examples\main\09_WavesPiston\wCasePistonBeach
DualSPHysics_v5.2\examples\main\09_WavesPiston\wCasePistonDamping
DualSPHysics_v5.2\examples\main\10_WavesPistonAWAS\wCasePistonDikeAWAS_REG
DualSPHysics_v5.2\examples\main\10_WavesPistonAWAS\wCasePistonWallAWAS_REG
DualSPHysics_v5.2\examples\inletoutlet\01_FlowCylinder
